### PR TITLE
Remove Faceted Admin Reports

### PR DESCRIPTION
**Pull request by esoergel**
Update:
Now ready to merge.  This replaces the User List report with https://github.com/dimagi/commcare-hq/pull/25636

Turns out there is a LOT of code connected to this stuff.  I basically started by removing the views, keeping track of utility functions that looked like they might be single-use.  Then I did that recursively for the utility functions.  I also searched for related functionality by grepping the diff for `reverse`,`url`, `include`, and `static`, and removed all of that stuff too.

@dannyroberts looks like you're gonna be scoping the ES upgrade, figured you'd be interested
@gbova @calellowitz buddies

##### PRODUCT DESCRIPTION
This replaces the old User List admin report with the new one introduced here: https://github.com/dimagi/commcare-hq/pull/25636
That will allow us to remove a lot of old admin functionality that was blocking much-needed upgrades.

### CODE REVIEW

**Comment by dannyroberts** [comment]
@esoergel This looks great. Keep me posted!

**Comment by snopoke** [comment]
@biyeun @esoergel what's the stats of this now?

**Comment by biyeun** [comment]
@snopoke I believe this was deprioritized in favor of other saas features... @dimagi/product

**Comment by esoergel** [comment]
Just added a replacement user list report which I hope to unblock this: https://github.com/dimagi/commcare-hq/pull/25636
Once that's merged, I'll rebase this PR and fix the conflicts.

**Comment by esoergel** [comment]
The new admin [user list report](https://github.com/dimagi/commcare-hq/pull/25636) is now live and ready to use.  This PR now switches references to the old user report to the new one and drops all the old code.  Should be good to go!

@djmore9 FYI

File: corehq/util/dates.py
Comment (orangejenny) [inline corehq/util/dates.py]: Nice attention to detail noticing this isn't used anymore.

**Review by orangejenny — APPROVED** [review]
This is a thing of beauty and a joy forever.

**Comment by orangejenny** [comment]
@esoergel I added one commit - aba3462f709 moves `nvd3_charts_helper` out of corehq now that it's only used in custom reports.

File: corehq/util/dates.py
Comment (orangejenny) [reply corehq/util/dates.py]: That is exactly what I do, too. Maybe Fowler will have better ideas (or we'll find someone has a tool/plugin for dead code).

Comment (orangejenny) [reply corehq/util/dates.py]: Looking around, maybe https://github.com/jendrikseipp/vulture ? I have not yet tried it.

Comment (esoergel) [reply corehq/util/dates.py]: I tried something like that (maybe that one, actually) a while back.  There were too many false positives to trust it, though I see now it has a `--min-confidence 100` option, which may help.

**Comment by biyeun** [comment]
yaaaaay! what a PR to come back from vacation to ❤️ 😻 🎉 🔥 ⚡️

**Pull request merged**
